components/panel/body-toggle.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import Dashicon from 'components/icon-button';
+
+function PanelBodyToggle( { opened, label, onClick } ) {
+	return (
+		<button className="components-panel__body-toggle" onClick={ onClick }>
```

Comment:
Regardless, we should do one of either use `Button` as a base (for consistency) or add `type="button"` to avoid this acting like a `submit` if ever used in a form.